dev/try_builders.json

```diff
@@ -181,6 +181,12 @@
          "enabled":true,
          "run_if":["examples/hello_world/**" ,"dev/**", "packages/flutter/**", "packages/flutter_driver/**", "packages/flutter_test/**", "packages/flutter_tools/lib/src/test/**", "packages/flutter_web_plugins/**", "bin/**"]
       },
+      {
+         "name":"Linux flutter_plugins",
+         "repo":"flutter",
+         "task_name":"flutter_plugins",
+         "enabled":true
```

Comment:
Do we want to run this builder on every pr or only when some paths of the code change? if we are interested in running only on subpaths I'd recommend to add a run_if property.

Reply:
Do we have some guidelines for how to decide whether to run a test on all PRs or not? I guess it's a cost/benefit tradeoff decision...
Technically any framework / tool changes or engine rolls can lead to this breaking (though limiting by these path probably covers almost all PRs).

WDYT?

Reply:
I don't think we have guidelines for that yet but let me add it to the engprod todo list. This is not a very expensive test and we can run it on every PR.